src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+$size-of-watch: 350px;
+$size-of-center: 0.06 * $size-of-watch;
+$first-color: rgba(30, 30, 30, 0.7);
+$second-color: rgba(255, 165, 0, 1);
+
+@mixin circle($size-of-watch) {
+  height: $size-of-watch;
+  width: $size-of-watch;
+  border-radius: 50%;
+}
+
+@keyframes spin {
```

Comment:
I suggest to use separated files and then import them inside the main.scss
```
styles
: main.scss
-- utils
    : _variables.scss
    : _extends.scss (for global styles)
-- blocks
    : clock.scss
```

Reply:
I will, this was just a template I guess, just wanted to commit so something is there.